### ISSUE TITLE
Entrypoint script error

### ISSUE THREAD

**Issue by webhive** [nginxinc/docker-nginx#414]
Discovered mainline/alpine/docker-entrypoint.sh and found some issues there
Tested on running alpine 3.7 container

```
# cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.7.0
PRETTY_NAME="Alpine Linux v3.7"
HOME_URL="http://alpinelinux.org"
BUG_REPORT_URL="http://bugs.alpinelinux.org"
```

These commands in ``docker-entrypoint.sh`` does not work as expected

```
/usr/bin/find /docker-entrypoint.d/ -type f -name "*.sh" -not -executable
find: unrecognized: -executable
BusyBox v1.27.2 (2018-06-06 09:08:44 UTC) multi-call binary.
```

```
/usr/bin/find "/docker-entrypoint.d/" -mindepth 1 -print -quit
find: unrecognized: -quit
BusyBox v1.27.2 (2018-06-06 09:08:44 UTC) multi-call binary.
```

So it definitely broken

**Comment by thresheek** [comment]
Indeed, those options are not available in alpine 3.7 (which is unsupported upstream anymore - so you definitely should upgrade!), alpine 3.8, alpine 3.9.

It works just fine with an updated busybox, at least on alpine 3.10 and 3.11.

**Comment by webhive** [comment]
Damn! Sorry - you are right. It was nginx 1.15 container which is sufficiently differ from master branch.
